sdk/servicebus/service-bus/package.json

```diff
@@ -46,18 +46,31 @@
     "@types/mocha": "^5.2.5",
     "@types/node": "^8.0.0",
     "@types/ws": "^6.0.1",
+    "assert": "^1.4.1",
     "buffer": "^5.2.1",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "cross-env": "^5.2.0",
     "delay": "^4.2.0",
     "dotenv": "^7.0.0",
     "https-proxy-agent": "^2.2.1",
+    "karma": "^4.0.1",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.2",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-junit-reporter": "^1.2.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-remap-coverage": "^0.1.5",
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi": "^1.0.1",
     "moment": "^2.24.0",
     "nyc": "^14.0.0",
+    "puppeteer": "^1.11.0",
```

Comment:
[nit] 
Looks like this is a manual addition.
Packages are not in alphabetical order.

Reply:
Also, I think we should be using rush to manage dependencies,
> `rush add -p puppeteer --dev` instead of `npm install puppeteer --save-dev`

https://rushjs.io/pages/commands/rush_add/

Reply:
That's ideal, or generally you can just add the dependency to the package.json manually and run `rush update`.

Reply:
Got it, I've been adding them manually. I've done a rush update and rearranged the scripts.